_user_guides/dinghy.md

```diff
@@ -290,3 +290,123 @@ The spinnaker `pipeline` stage allows you to trigger other pipelines. However, t
 {% endraw %}```
 
 In the above example, we are triggering a pipeline by the name `default-pipeline` under `default-app` spinnaker application. The app name and the pipeline name can be overwritten when calling this module. At any higher level, simply pass in `"triggerApp"` and `"triggerPipeline"` like so: `{% raw %}{{ module "pipeline.stage.module" "triggerApp" "pipelineidtest" "triggerPipeline" "testpipelinename" }}{% endraw %}`
+
+
+## Advanced features:
+
+### Monorepo
+Dinghy supports multiple spinnaker applications under the same git repo. eg:
+```{% raw %}
+monorepo/
+├── app1
+│   ├── bin
+│   ├── lib
+│   ├── pkg
+│   └── src
+│       ├── app1.go
+│       └── dinghyfile
+└── app2
+    ├── bin
+    ├── lib
+    ├── pkg
+    └── src
+        ├── app2.go
+        └── dinghyfile
+{% endraw %}```
+
+Notice both `app1` and `app2` are under the same repo, each app has its own `dinghyfile` and its own spinnaker application that can be referenced in the `dinghyfile`.
+
+### Template validation
+If while rendering a `dinghyfile`, there is a malformed JSON filed that is encountered, the logs should indicate the line number and the column number of the error. There is also the `arm cli` that can be used to validate `dinghyfile`s and `module`s locally without having to put them in source control.
```

Comment:
"If, while rendering a 'dinghyfile', a malformed JSON file is encountered, the logs..." (missing comma after "If", remove "there is" and "that", and take the d off "filed" (typo)).

Also, "The `arm cli` command can also be used to validate..."